src/lib/traceTypes.js

```diff
@@ -33,6 +29,10 @@ export const chartCategory = _ => {
       label: _('3D'),
       maxColumns: 1,
     },
+    HIDDEN: {
+      value: 'HIDDEN',
+      label: '',
```

Comment:
so because it does not have a label its going to be hidden?

Reply:
no, it's hidden because it's not included in the next array